tools/pyboard.py

```diff
@@ -359,6 +360,41 @@ def follow(self, timeout, data_consumer=None):
         # return normal and error output
         return data, data_err
 
+    def raw_paste_write(self, command_bytes):
+        # Read initial header, with window size.
+        data = self.serial.read(2)
+        window_size = data[0] | data[1] << 8
```

Comment:
Would it be "more pythonic" to use something like `struct.unpack` here? :wink:

Reply:
I try not to depend on entire modules if it's just for a very simple thing like this... too much micro, not enough Python :)